database/migrations/001_init_database.go

```diff
@@ -120,8 +120,9 @@ var Migration001InitDatabase = &migrate.Migration{
 			builder_pubkey  varchar(98) NOT NULL,
 			description    	text NOT NULL,
 
-			is_high_prio    boolean NOT NULL,
-			is_blacklisted  boolean NOT NULL,
+			status             int NOT NULL,
+			collateral_value   NUMERIC(48, 0),
+			collateral_id      bigint NOT NULL,
```

Comment:
Suggest making `collateral_id` a string so that relay operators can set it to be a human-readable string (e.g. "builder0x69", "flashbots").

Reply:
sure! done!